packages/cli/package.json

```diff
@@ -49,8 +49,10 @@
     "@vercel/python": "3.1.33",
     "@vercel/redwood": "1.0.44",
     "@vercel/remix": "1.1.6",
+    "@vercel/routing-utils": "workspace:2.1.3",
```

Comment:
Is `workspace:` necessary here?

Reply:
Ahh thats a pnpm thing. https://pnpm.io/cli/add#install-from-the-workspace

Might have to update all them to use this `workspace:` syntax

Reply:
Related/unrelated, FWIW when you create a new Turbo repo, the template uses `workspace:`: https://github.com/vercel/turbo/blob/main/examples/basic/packages/ui/package.json#L14.

Reply:
Yeah that seems correct for `turbo`. However, does `learna` know how to handle `workspace:`?

Reply:
https://lerna.js.org/docs/recipes/using-pnpm-with-lerna it supports it

Reply:
It does in v5! Migrating all workspace packages to the prefix

Reply:
I'm curious how this works with the published package. Does lerna change the version numbers when publishing?